src/search/result.ts

```diff
@@ -120,10 +121,12 @@ export class IndexedDoc<T extends TigrisCollectionType> {
 export class DocMeta {
```

Comment:
Is this same as what we have for collection's search? if not I'd suggest to consolidate.

Reply:
Both Collection/Index search have identical response

Reply:
Okay, so DocMeta is used by both?